CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+X.Y.Z Release notes
+=============================================================
+### Breaking changes
+* None.
+
+### Enhancements
+* None
+
+### Bug fixes
+* Spurious closing of sync session (#1391).
+* Custom path not respected (#1392 and #1393).
```

Comment:
Again, this should be more explicit about what's being fixed. "Ensure that a custom path is respected when using `Realm.open`"